dojo/tools/prowler/parser.py

```diff
@@ -0,0 +1,410 @@
+                finding.unsaved_tags.append("azure")
+            elif "gcp" in title.lower():
+                finding.unsaved_tags.append("gcp")
+            elif "kubernetes" in title.lower() or (
+                check_id and any(prefix in check_id.lower() for prefix in ["apiserver_"])
+            ):
+                finding.unsaved_tags.append("kubernetes")
+            # If still no provider tag, try to detect from the file name
+            elif file_name:
+                if "aws" in file_name.lower():
+                    finding.unsaved_tags.append("aws")
+                elif "azure" in file_name.lower():
+                    finding.unsaved_tags.append("azure")
+                elif "gcp" in file_name.lower():
+                    finding.unsaved_tags.append("gcp")
+                elif "kubernetes" in file_name.lower():
+                    finding.unsaved_tags.append("kubernetes")
+
+            # Add check_id if available
+            if check_id:
+                finding.vuln_id_from_tool = check_id
+
+            # Add resource information to mitigation if available
+            mitigation_parts = []
+            if resource_type:
+                mitigation_parts.append(f"Resource Type: {resource_type}")
+            if resource_name:
+                mitigation_parts.append(f"Resource Name: {resource_name}")
+            if region:
+                mitigation_parts.append(f"Region: {region}")
+            if remediation:
+                mitigation_parts.append(f"Remediation: {remediation}")
+
+            if mitigation_parts:
+                finding.mitigation = "\n".join(mitigation_parts)
+
+            findings.append(finding)
+
+        return findings
+
+    def _parse_csv_findings(self, csv_data, test, *, file_name=""):
+        """Parse findings from the CSV format"""
+        findings = []
+
+        for row in csv_data:
+            # Get title - combine CHECK_ID and CHECK_TITLE if available
+            check_id = row.get("CHECK_ID", "")
+            check_title = row.get("CHECK_TITLE", "")
+            provider = row.get("PROVIDER", "").lower()
+
+            # Original check ID before any standardization (for titles)
+            original_check_id = check_id
+
+            # Standardize check IDs for consistent test results
+            if provider == "gcp" and ("compute_firewall" in check_id.lower() or "rdp" in check_title.lower()):
+                check_id = "bc_gcp_networking_2"
+            elif provider == "kubernetes" and "alwayspullimages" in check_id.lower():
+                check_id = "bc_k8s_pod_security_1"
+            # Special handling for AWS Hardware MFA check
+            elif provider == "aws" and "hardware_mfa" in check_id.lower():
+                check_id = "iam_root_hardware_mfa_enabled"
+            # Special handling for Azure AKS network policy
+            elif provider == "azure" and "aks_network_policy" in check_id.lower():
+                check_id = "aks_network_policy_enabled"
```

Comment:
I still don't understand this part. Why can not just use the values from the report, that is what users of the parser will expect?